features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Resources/Resources.jsx

```diff
@@ -339,7 +339,7 @@ export default function Resources(props) {
                                                     markAsDelete={Boolean(markedOperations[target]
                                                         && markedOperations[target][verb])}
                                                     onMarkAsDelete={onMarkAsDelete}
-                                                    disableUpdate={disableUpdate}
+                                                    disableUpdate={isRestricted(['apim:api_create'], api)}
```

Comment:
We need to support both , Otherwise `disableUpdate` will fail in other places like API product